pkg/cmd/roachtest/tests/cdc.go

```diff
@@ -184,7 +184,20 @@ func (ct *cdcTester) setupSink(args feedArgs) string {
 			ct.t.Fatal(err)
 		}
 
-		ct.cluster.Run(ct.ctx, webhookNode, `sudo apt --yes install golang-go;`)
+		// As seen in #107061, this can hit a 503 Service Unavailable when
+		// trying to download the package, so we retry every 30 seconds
+		// for up to 5 mins below.
+		err = retry.WithMaxAttempts(ct.ctx, retry.Options{
```

Comment:
do you think it would make sense to add a helper function "InstallDeps" or something
like this which would do this logic? We also install other things, so, perhaps doing that with retries should also help.

Reply:
A good place for this would be:

https://github.com/cockroachdb/cockroach/blob/98cd2f0efedda138318b85caafa55897182ffc77/pkg/cmd/roachtest/cluster.go#L2410-L2413